spid_cie_oidc/relying_party/views/rp_extend_session.py

```diff
@@ -0,0 +1,127 @@
+import logging
+
+import requests
+from django.contrib.auth.models import User
+from djagger.decorators import schema
+from django.conf import settings
+from django.contrib.auth import logout
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from django.utils import timezone
+from spid_cie_oidc.entity.jwtse import create_jws
+from spid_cie_oidc.entity.models import FederationEntityConfiguration
+from ..models import OidcAuthenticationToken
+from ..oauth2 import *
+from ..oidc import *
+
+from spid_cie_oidc.entity.jwtse import (
+    unpad_jwt_payload,
+    verify_jws,
+    verify_at_hash
+)
+
+logger = logging.getLogger(__name__)
+
+
+@schema(
+    summary="OIDC Relying party refresh token request",
+    methods=['GET'],
+    external_docs = {
+        "alt_text": "AgID SPID OIDC Guidelines",
+        "url": "https://www.agid.gov.it/it/agenzia/stampa-e-comunicazione/notizie/2021/12/06/openid-connect-spid-adottate-linee-guida"
+    },
+    tags = ['Relying Party']
+)
+
+@login_required
+def oidc_rp_extend_session(request):
```

Comment:
the code below seems to be replicated from the one we already have in the RP class

I'd suggest to not duplicate code but to integrate this in the preexisting RP class

do you agree @rglauco ?